### ISSUE TITLE
Remove venue: text on home screen

### ISSUE THREAD

**Issue by abhijeetmanhas** [KamandPrompt/SNTC-Calendar#38]
Sometimes it can be just a link or description, so venue written there looks odd.

**Comment by mayankjainllrl** [comment]
Can I work on this?

**Comment by abhijeetmanhas** [comment]
> Can I work on this?

yes sure! https://activities.iitmandi.co.in this is the website, in your PR, post screenshot as well.
